### PR TITLE
XXL grid size

### PR DESCRIPTION
**Pull request by baebb**
XXL grid size for large screens

new size: 1520px

### CODE REVIEW

**Comment by cvrebert** [comment]
Approximate duplicate of #14370.

**Comment by cvrebert** [comment]
Why 1520px specifically?

File: scss/_variables.scss [hunk 1]
```diff
@@ -123,7 +123,8 @@ $grid-breakpoints: (
   sm: 544px,
   md: 768px,
   lg: 992px,
-  xl: 1200px
+  xl: 1200px,
+  xxl: 1560px
```
Comment (cvrebert) [inline scss/_variables.scss]: I presume this is a typo and you meant `1520px` here?

Comment (justrossthings) [inline scss/_variables.scss]: yeah, oops

**Comment by justrossthings** [comment]
i picked 1520 cause it's 1.3x1200 and 992 is 1.3x768.

is there a better number to choose before i fix the 1560px mistake?

**Comment by cvrebert** [comment]
@mdo Has your opinion changed since https://github.com/twbs/bootstrap/issues/14370#issuecomment-51784874 ?

**Comment by justrossthings** [comment]
@mdo @cvrebert is it problematic to have >4 grid sizes? 

I get that it's not hard to change the variables to your requirements via SASS and recompile, but me 3yrs ago [and other fresh devs] wouldn't have been able to do this. The point of BS was to make great web design accessible, right?

If there's a good reason to keep it 4 sizes then standard breakpoints should be updated at least.

**Comment by mdo** [comment]
Going to continue to pass on this. As this PR demonstrates, it's very easy to add a new tier to the grid (and get all the classes and mixins for free). We'll keep it as-is for now.

**Comment by cvrebert** [comment]
> is it problematic to have >4 grid sizes?

(Actually it'd be >5 since there are already 5 sizes.)
Not inherently, but adding sizes has trade-offs. Each additional grid size increases file size/"bloat" and increases cognitive load slightly. (And eventually, the names for the sizes start to get uglier/weirder.) So we have to balance these costs against how common and useful each new grid size would be. It's ultimately a judgment call.

> then standard breakpoints should be updated at least.

If you have specific suggestions, feel free to open a new issue. :smile:

**Pull request closed without merging**